playground/backend/api.js

```diff
@@ -56,7 +69,82 @@ async function doFetch(url) {
       'Referer': 'https://amp.dev/playground',
```

Comment:
Wondering whether it'd be better to use a mobile user agent.

Reply:
`Android`, `Nexus` and `Mobile` should be enough hints for most user-agent tests to count as mobile, no? Just wondering if we should update versions to prevent getting a "update your browser" page.